homeassistant/components/lunatone/light.py

```diff
@@ -0,0 +1,108 @@
+"""Platform for Lunatone light integration."""
+
+from __future__ import annotations
+
+from typing import Any
+
+from homeassistant.components.light import ColorMode, LightEntity
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddConfigEntryEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import LunatoneConfigEntry, LunatoneDevicesDataUpdateCoordinator
+
+PARALLEL_UPDATES = 0
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: LunatoneConfigEntry,
+    async_add_entities: AddConfigEntryEntitiesCallback,
+) -> None:
+    """Set up the Lunatone Light platform."""
+    coordinator_info = config_entry.runtime_data.coordinator_info
+    coordinator_devices = config_entry.runtime_data.coordinator_devices
+
+    info_api = coordinator_info.info_api
+    devices_api = coordinator_devices.devices_api
+
+    async_add_entities(
+        [
+            LunatoneLight(coordinator_devices, device.id, info_api.serial_number)
```

Comment:
lets use `coordinator_info.data.device.serial` instead of `info_api`, since `info_api.serial_number` can be `None`